Marlin/Marlin_main.cpp

```diff
@@ -5286,12 +5294,32 @@ void home_all_axes() { gcode_G28(true); }
         // Probe the points
 
         if (!_7p_half_circle && !_7p_triple_circle) { // probe the center
-          z_at_pt[0] += probe_pt(dx, dy, stow_after_each, 1, false);
+          #if !ENABLED(PROBE_MANUALLY)
+            z_at_pt[0] += probe_pt(dx, dy, stow_after_each, 1, false);
+          #else
+            _man_probe_pt(0, 0);
+            KEEPALIVE_STATE(PAUSED_FOR_USER);
+            wait_for_user = true;
+            while (wait_for_user) idle();
+            wait_for_user = false;
```

Comment:
Since `while(wait_for_user)` just finished, we know `wait_for_user` is already `false`.